src/_includes/toc-and-content.html

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+  <div class="toc col-md-3">
+    <div class="toc-container">
+      <div class="d-none d-sm-none d-md-block d-lg-block">
+        <h4>Table of Contents</h4>
```

Comment:
These heading levels in the TOC need some work:

* Currently the headings jump from `<h1>How to Go Contactless</h1>` to this `<h4>Table of Contents</h4>`
* The table of contents wrapper `.toc` is a sibling to the `.content-container` element, so they should have the same heading level order / restart with their own headings
* The first heading under `Table of Contents` should step down a level, not be at the same level as the TOC heading.

Requested changes:

* `<h2>Table of Contents</h2>`
* `<h3>{{ item.title }}</h3>`
* I don't think it is correct that each `<li><a>` is wrapped in a heading, these are just jump links and don't mark out a new section of content - we don't want to use headings for _style_ but for semantic structure.

Reply:
Nice!